email_from_template/send_mail.py

```diff
@@ -74,6 +74,20 @@
     </a>
     {% endblock %}
 
+:mod:`django-email-from-template` also has the ability to only send the email
+when the current transaction commits. Pass ``on_commit=True`` to ``send_mail``
+to enable this functionality - for backwards compatability and predictability
+it is not enabled by default. Note that the rendering still occurs during the
+transaction, it is only the sending that is deferred::
```

Comment:
>  only the sending that is deferred

Indeed which is pretty nice, ie. nicer than it failing to render post-commit. Otherwise we would just call email-from-template's ``send_mail`` using ``transaction.post_commit``. Can you document...?